00_ES6/05_redux_selector/README.md

```diff
@@ -0,0 +1,397 @@
+# Intro
+
+In this new sample, let's demonstrate the value of using `reselect` as an interface to access our redux store in a memoized fashion. With memoized selectors we can compute derived data from the store in a very efficient way as every result is cached (selectors are not recomputed unless one of its arguments changes). This means, sucessive calls to the same selector with the same arguments are served immediately. Great for expensive calls, right?
+
+For this sample, we are going to add a new functionality to our chat: message search. For that purpose, we must create a few things first.
+
+You can start from the previous sample: 04_redux_saga.
+
+# Steps
+
+- Let's create a new component to search in our chat log. You can add the following new files:
+
+_./src/pages/chat/components/chat-search.component.jsx_
+```diff
```

Comment:
If it's a new component, instead of _diff_ use _javascript_

Reply:
Please apply this to all code in readme, thumb rule:

- If it's new file, use always javascript, no diff (sintax highlight).
- If it's an update on a file (remove lines, add line), use diff

Reply:
Done. fixed in several places and committed.